### DISCUSSION TITLE
getting started with using my own images as input.

### DISCUSSION THREAD

**Discussion by gschian0** [royorel/StyleSDF#33] [Q&A]
Can you please point me to where I can get started learning how to use my own images for this?  I am not a newbie but am fresh and would like to play around with this for some artistic projects.  :)

**Comment by royorel** [comment]
Hi @gschian0 ,

I'm not sure I understand your question. Do you mean training the network on your own data? Or using a specific image as input during test time and project it to the Generator latent space?

If it's the former, please follow the instructions in the readme page. If it's the latter, unfortunately the code does not support latent space projection.

**Reply by gschian0** [reply]
Well, first I am doing a series of self portraits.  I would like to send my face as the input and render some videos with the output mesh.  :) Thankyou for the quick response.

**Reply by royorel** [reply]
So it seems like you're interested in the latter case. Unfortunately, our code does not support this.
